modules/collabora_online_group/src/Plugin/Group/RelationHandler/CollaboraPermissionProvider.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\collabora_online_group\Plugin\Group\RelationHandler;
+
+use Drupal\groupmedia\Plugin\Group\RelationHandler\GroupMediaPermissionProvider;
+
+/**
+ * Provides Collabora permissions for group.
+ */
+class CollaboraPermissionProvider extends GroupMediaPermissionProvider {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildPermissions(): array {
+    $permissions = $this->parent->buildPermissions();
+
+    /* @see \Drupal\group\Plugin\Group\RelationHandlerDefault\PermissionProvider::buildPermissions() */
+    $provider_chain = $this->groupRelationTypeManager()->getPermissionProvider($this->pluginId);
+
+    // Add Collabora permissions.
+    $prefix = 'Entity:';
+    if ($name = $provider_chain->getPermission('preview in collabora', 'entity')) {
+      $permissions[$name] = $this->buildPermission("$prefix Preview %entity_type in collabora");
+    }
+    if ($name = $provider_chain->getPermission('edit in collabora', 'entity')) {
+      $permissions[$name] = $this->buildPermission("$prefix Edit any %entity_type in collabora");
+    }
+    if ($name = $provider_chain->getPermission('edit in collabora', 'entity', 'own')) {
+      $permissions[$name] = $this->buildPermission("$prefix Edit own %entity_type in collabora");
+    }
+
+    return $permissions;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getPermission($operation, $target, $scope = 'any'): bool|string {
+    if ($target === 'entity') {
+      switch ($operation) {
+        case 'preview in collabora':
+          return "preview $this->pluginId in collabora";
+
+        case 'edit in collabora':
+          if (
+            $this->definesEntityPermissions &&
+            ($this->implementsOwnerInterface || $scope === 'any')
+          ) {
```

Comment:
In core you will find the following pattern for multi-line conditions:
```suggestion
          if ($this->definesEntityPermissions
            && ($this->implementsOwnerInterface || $scope === 'any')
          ) {
```
Is this a convention? 
Not yet.
In core you find both variations (&& at end of line and at beginning of line).
There is a discussion here, https://www.drupal.org/project/coding_standards/issues/3392321

You can use regex search like `^  +(&&|\|\|)` and `(&&|\|\|)$` to find examples for the two patterns.

Sorry I did not see this in the first review.

Reply:
Personally I prefer end of the condition, matter of symmetry when reading, first condition will be always "unaligned". Open to discussion, we can agree all together about establishing a standard.

Reply:
Ok, thiis can remain as is. Not yet sure if this is the final convention but at least for here let's not block the PR.